taxonomies/ingredients.txt

```diff
@@ -23355,7 +23395,7 @@ fr:blé amidonnier, amidonnier
 ru:двузернянка
 
 <en:cereal
-en:teff
+en:teff, teff seeds
 es:teff, semilla de teff
```

Comment:
I prefer to keep the seeds separate. Just `teff` does not say anything about form, shape or process.

(but maybe I am exaggerating)

Reply:
I actually looked it up on wikipedia at the time, and that says "It is cultivated for its edible seeds, also known as teff", so I just put them together. Presumably any use of the term in ingredients lists would be grain rather than the plant anyway? i dunno. :)

Reply:
I would make `teff seeds` a child of `teff`, and `teff flour` a child of `teff`, and `teff seed flour`, a child of `teff seed` and `teff flour`.

Reply:
I went through the CSV to see what other terms there were, and just confused myself trying to decide a hierarchy, so I just made them all children of teff.